src/pages/ResourceForm.tsx

```diff
@@ -196,8 +197,18 @@ const ResourceForm: FC<ResourceFormProps> = ({ uppy, resource, resourceType }) =
                 <StyledPanel>
```

Comment:
Kunne du laget en fil for innmaten til "fanen" ?

Reply:
Fikset